pkg/controller/nodelifecycle/node_lifecycle_controller.go

```diff
@@ -223,7 +222,7 @@ type Controller struct {
 	taintNodeByCondition bool
 
 	nodeUpdateChannels []chan *v1.Node
-	nodeUpdateQueue    workqueue.Interface
+	nodeUpdateQueue    *cache.FIFO
```

Comment:
I'm fine with trying that, but please add a comment so that we know what this was changed.